lib/tasks/zomato_api.rake

```diff
@@ -0,0 +1,81 @@
+require 'net/http'
+require 'uri'
+require 'json'
+
+namespace :zomato_api do
+
+  def getResponse uri
```

Comment:
always wrap the arguments of a method with parenthesis in the method definition.
`getResponse(uri)`

Reference: https://github.com/rubocop-hq/ruby-style-guide#method-parens